ignite/contrib/handlers/tqdm_logger.py

```diff
@@ -172,19 +168,16 @@ def attach(self, engine, metric_names=None, output_transform=None,
         """
         desc = self.tqdm_kwargs.get("desc", "Epoch")
 
-        if not isinstance(event_name, (Events, EventWithFilter)):
-            raise ValueError("Logging event should be only `ignite.engine.Events`")
-
-        if isinstance(closing_event_name, EventWithFilter):
```

Comment:
This condition is necessary for closing events. We should not close progress bar with some event filtering, IMO...

Reply:
Okay, then we can use a check for the type of filter. I'll add this